flake.nix

```diff
@@ -66,49 +76,28 @@
               inherit (pkgs.python3Packages)
                 python
                 click;
-            }
-          ++ {
-            "x86_64-linux" = [ (pkgs.callPackage ./arm-gnu-gcc.nix { }) ];
-            "aarch64-linux" = [ (pkgs.gcc13.override { propagateDoc = true; isGNU = true; }) ];
-            "aarch64-darwin" = [ ];
```

Comment:
I'm confused by this. 
on aarch64-linux, didn't we install gcc from nix before? Now we do not anymore?

Reply:
Before I used `pkgs.mkShellNoCC` to build to nix shell, which explicitly excluded nix CC in the shell. But now I changed it to `pkgs.mkShell` which packed gcc and configured it in nix shell by default, so that I don't have to do the similar trick as x86_64 for aarch64.

Reply:
Oh, turns out I just have to add the specific gcc into the dependency then we're all fine

Reply:
Thanks for the catch, it will be less confusing to add the specific gcc version in the shell

Reply:
Thanks!